Formula/qwt.rb

```diff
@@ -41,11 +42,10 @@ def install
     on_linux do
       spec = "linux-g++"
     end
-    spec << "-arm64" if Hardware::CPU.arm?
     args << spec
 
-    qt5 = Formula["qt@5"].opt_prefix
-    system "#{qt5}/bin/qmake", *args
+    qt = Formula["qt"].opt_prefix
+    system "#{qt}/bin/qmake", *args
```

Comment:
```suggestion
    qt = Formula["qt"]
    system "#{qt.opt_prefix}/bin/qmake", *args
```
This way we don't need to keep repeated `Formula["qt"]` below and just use `qt`.